### PR TITLE
[AI Training guides] - modify folders name + add new guide for billing

### PR DESCRIPTION
**Pull request by eleapttn**
1. Folders name modification:

- change the number only in the folders name (`training_guide_01` -> `training_guide_08`)
- **NO** changes in the slugs

2. Add a new AI Training guide:

- create the `guide.en-gb.md` file in order to explain how we bill the AI Training product

### CODE REVIEW

File: pages/platform/ai/training_guide_08_billing_concept/guide.en-gb.md [hunk 1]
```diff
@@ -0,0 +1,109 @@
...
+![image](images/ai.training.items.png){.thumbnail}
+
+A more detailed view:
+
+![image](images/ai.training.billing.png){.thumbnail}
+
+### Compute resources details
+
+During the AI Training job creation, you can select **compute resources**, known as CPUs or GPUs.
+Their official pricing is available in the [OVHcloud Control Panel](https://www.ovh.com/auth/?action=gotomanager&from=https://www.ovh.co.uk/&ovhSubsidiary=GB) or on the[OVHcloud Public Cloud website](https://www.ovhcloud.com/en-gb/public-cloud/prices/).
```
Comment (philippart-s) [inline pages/platform/ai/training_guide_08_billing_concept/guide.en-gb.md]:
```suggestion
Their official pricing is available in the [OVHcloud Control Panel](https://www.ovh.com/auth/?action=gotomanager&from=https://www.ovh.co.uk/&ovhSubsidiary=GB) or on the [OVHcloud Public Cloud website](https://www.ovhcloud.com/en-gb/public-cloud/prices/).
```

File: pages/platform/ai/training_guide_08_billing_concept/guide.en-gb.md [hunk 2]
```diff
@@ -0,0 +1,109 @@
...
+
+**Last updated 14th February, 2023.**
+
+## Objective
+
+The OVHcloud AI Training service provides you a container as a service platform, linked to CPU or GPU resources, without the hassle of installing or operating them. This guide will cover the lifecycle of an AI Training job, and associated billing.
+
+## Introduction
+
+AI Training jobs are linked to a Public Cloud project. The whole project is billed at the end of the month, with pay-as-you-go. It means you will only pay for what you consume, based on the compute resources you use (CPus and GPUs) and their running time.
```
Comment (MathieuBsqt) [inline pages/platform/ai/training_guide_08_billing_concept/guide.en-gb.md]:
```suggestion
AI Training jobs are linked to a Public Cloud project. The whole project is billed at the end of the month, with pay-as-you-go. It means you will only pay for what you consume, based on the compute resources you use (CPUs and GPUs) and their running time.
```

File: pages/platform/ai/training_guide_08_billing_concept/guide.en-gb.md [hunk 3]
```diff
@@ -0,0 +1,109 @@
...
+
+## AI Training job lifecycle
+
+During its lifetime, the [AI Training job](https://docs.ovh.com/gb/en/publiccloud/ai/training/jobs/) will go through the following statuses:
+
+- `QUEUED`: the job run request is about to be processed.
+- `INITIALIZING`: the job instance is created and the data is synchronized from the Object Storage. To know more about the data synchronisation check out the [Data How it works](https://docs.ovh.com/gb/en/publiccloud/ai/data/#how-it-works) section.
+- `PENDING`: the job is being started.
+- `RUNNING`: the job is running, you can connect to it. Compute resources (GPUs/CPUs) are allocated to your specific job and data are available.
+- `INTERRUPTING`: the job is still running but an interruption order was received and is about to be processed
```
Comment (MathieuBsqt) [inline pages/platform/ai/training_guide_08_billing_concept/guide.en-gb.md]:
```suggestion
- `INTERRUPTING`: the job is still running but an interruption order was received and is about to be processed.
```

File: pages/platform/ai/training_guide_08_billing_concept/guide.en-gb.md [hunk 4]
```diff
@@ -0,0 +1,109 @@
...
+When working with remote data, you pay separately for the storage of this data.
+The pricing of Object Storage is apart from the AI Training pricing.
+
+### Pricing examples
+
+#### Example 1: one CPU job for 10 days
+
+We start one AI Training job, with two CPUs and we keep it running for 10 days then it is stopped.
+
+- compute resources: 2 x CPU (0,03€ / hour)
```
Comment (MathieuBsqt) [inline pages/platform/ai/training_guide_08_billing_concept/guide.en-gb.md]:
```suggestion
- compute resources: 2 x CPU (0.03€ / hour)
```

File: pages/platform/ai/training_guide_08_billing_concept/guide.en-gb.md [hunk 5]
```diff
@@ -0,0 +1,109 @@
...
+- remote storage:  nothing
```
Comment (MathieuBsqt) [inline pages/platform/ai/training_guide_08_billing_concept/guide.en-gb.md]:
```suggestion
- remote storage: nothing
```

File: pages/platform/ai/training_guide_08_billing_concept/guide.en-gb.md [hunk 6]
```diff
@@ -0,0 +1,109 @@
...
+- duration: 10 days then stopped
+
+Price calculation for compute: 10 (days) x 24 (hours) x 2 (CPU) x 0,03€ (price / CPU) = **14,40 euros**, billed at the end of the month
```
Comment (MathieuBsqt) [inline pages/platform/ai/training_guide_08_billing_concept/guide.en-gb.md]:
```suggestion
Price calculation for compute: 10 (days) x 24 (hours) x 2 (CPU) x 0.03€ (price / CPU) = **14.40 euros**, billed at the end of the month
```

File: pages/platform/ai/training_guide_08_billing_concept/guide.en-gb.md [hunk 7]
```diff
@@ -0,0 +1,109 @@
...
+
+#### Example 2: one GPU job for 5 hours
+
+We start one AI Training job with two GPUs and we keep it running for 5 hours then it is stopped.
+
+- compute resources: 2 x GPU NVIDIA V100s (1,93€ / hour)
```
Comment (MathieuBsqt) [inline pages/platform/ai/training_guide_08_billing_concept/guide.en-gb.md]:
```suggestion
- compute resources: 2 x GPU NVIDIA V100s (1.93€ / hour)
```

File: pages/platform/ai/training_guide_08_billing_concept/guide.en-gb.md [hunk 8]
```diff
@@ -0,0 +1,109 @@
...
+- remote storage: nothing
+- duration: 5 hours then stopped
+
+Price calculation for compute : 5 (hours) x 2 (GPU) x 1,93€ (price / GPU) = **9,65 euros**, billed at the end of the month
```
Comment (MathieuBsqt) [inline pages/platform/ai/training_guide_08_billing_concept/guide.en-gb.md]:
```suggestion
Price calculation for compute : 5 (hours) x 2 (GPU) x 1.93€ (price / GPU) = **9.65 euros**, billed at the end of the month
```

**Review by MathieuBsqt — COMMENTED** [review]
Small typo fix suggestions

**Comment by eleapttn** [comment]
🚀

**Comment by eleapttn** [comment]
Information for doc duplicates: 

- Title FR: Facturation et cycle de vie (EN)
- Excerpt FR: Découvrez comment sont facturés les jobs AI Training

Thanks to the team for the duplication 😉

**Comment by OVH-Team-Guides** [comment]
Thank you for your contribution @eleapttn

**Pull request merged**